Scripts/script-ContextGetEmails.yml

```diff
@@ -0,0 +1,45 @@
+commonfields:
+  id: ContextGetEmails
+  version: -1
+name: ContextGetEmails
+script: |+
+  if (args.domains)
+      var domains = args.domains.split(',');
```

Comment:
domain should be defined outside the condition, it is bad practice to have it like that